filebeat/cmd/root.go

```diff
@@ -33,12 +33,10 @@ import (
 )
 
 // Name of this beat
-var Name = "filebeat"
+const Name = "filebeat"
 
-// RootCmd to handle beats cli
-var RootCmd *cmd.BeatsRootCmd
-
-func init() {
+// Filebeat build the beat root command for executing filebeat and it's subcommands.
+func Filebeat(inputs beater.PluginFactory) *cmd.BeatsRootCmd {
```

Comment:
I am on the fence about this change. The code itself is better. I am just a bit disappointed that the consistency over different beats `cmd/root.go` files are gone. But I guess I can live with that. :D

Reply:
yeah, don't like it either. This last integration step is kinda ugly TBH. But trying to not use globals and x-pack filebeat relying on globals kinda forced me to do this. Long term I think `libbeat/cmd` and the `<beats>/cmd` packages will be removed.